phpBB/includes/ucp/ucp_pm_viewfolder.php

```diff
@@ -104,6 +105,20 @@ function view_folder($id, $mode, $folder_id, $folder)
 			'S_MOVE_MARKED_OPTIONS'	=> $s_folder_move_options)
 		);
 
+		// Get avatars
+		$avatars = array();
+		$sql = 'SELECT *
+					FROM ' . USERS_TABLE ;
```

Comment:
You should get the list of user_ids that are used and limit the query. This query will take too long for big boards.